polymetis/polymetis/include/polymetis/polymetis_server.hpp

```diff
@@ -29,6 +29,7 @@
 #define MAX_MODEL_BYTES 1048576         // 1 megabyte
 #define THRESHOLD_NS 1000000000         // 1s
 #define SPIN_INTERVAL_USEC 20000        // 0.02s (50hz)
+#define NON_RT_PRIO 40
```

Comment:
Suggest rename to RT_LOW_PRIO.
Because it is still a realtime process as long as your scheduler is FIFO or RR.
Non realtime is OTHER/NORMAL.